TopMovies/API/MovieAPI.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  MovieAPI.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 02.02.2021.
+//
+
+import Foundation
+
+protocol MovieAPIProtocol {
+    func getMovies()
+}
+
+struct MovieAPI: MovieAPIProtocol {
+    func getMovies() {
```

Comment:
`getMovies` -> `moviesList`
Let's avoid `get` since we know that API should return something to us.